src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -251,6 +269,10 @@
         (time! (:package-gc-time admin-metrics)
                (gc-packages! db))
         (counters/inc! (:package-gcs admin-metrics)))
+      (when (request "gc_resource_events")
```

Comment:
The analogous request for deleting reports is `purge_reports` should this be `purge_resource_events`?